ecosystem/sep-0012.md

```diff
@@ -7,7 +7,7 @@ Author: Interstellar
 Status: Accepted
 Created: 2018-09-11
 Updated: 2018-09-11
-Version 1.0.0
+Version 2.0.0
```

Comment:
Since we're keeping the changes backwards-compatible, do we need to update the major version? Or should we use `1.1.0` instead?

Reply:
Yep, your'e right. Done.